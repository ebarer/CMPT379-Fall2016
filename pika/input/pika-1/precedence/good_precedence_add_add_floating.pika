exec {

  print 3.4 + 9.4 + 7.4, _n_.
  print 3.9 + 2.0 + 5.4, _n_.
  print 8.7 + 6.2 + 6.4, _n_.
  print 7.4 + 2.9 + 9.7, _n_.
  print 5.5 + 7.5 + 1.0, _n_.
  print 1.6 + 5.6 + 4.1, _n_.
  print 4.6 + 9.9 + 7.0, _n_.
  print 2.9 + 9.0 + 5.8, _n_.
  print 8.1 + 7.9 + 6.8, _n_.
  print 3.1 + 7.2 + 6.8, _n_.
  print 7.2 + 7.1 + 8.3, _n_.
  print 2.6 + 1.1 + 6.8, _n_.

  print 4.2 + 7.5 + 4.6, _n_.
  print 7.5 + 3.8 + 6.3, _n_.
  print 4.7 + 7.8 + 9.4, _n_.
  print 9.3 + 3.1 + 9.0, _n_.
  print 6.8 + 3.7 + 6.8, _n_.
  print 2.2 + 6.2 + 1.3, _n_.
  print 7.9 + 1.6 + 5.7, _n_.
  print 7.5 + 4.2 + 5.6, _n_.
  print 5.9 + 6.9 + 3.3, _n_.
  print 3.0 + 2.9 + 2.9, _n_.
  print 5.5 + 7.4 + 4.4, _n_.
  print 2.9 + 2.3 + 9.1, _n_.

  print 2.9 + 1.7 + 5.7 + 5.8, _n_.
  print 4.2 + 5.8 + 5.0 + 8.7, _n_.
  print 7.3 + 3.5 + 6.6 + 1.8, _n_.
  print 1.3 + 6.1 + 4.0 + 3.3, _n_.
  print 9.3 + 2.4 + 1.5 + 2.9, _n_.
  print 8.1 + 8.5 + 3.0 + 7.2, _n_.
  print 7.6 + 2.4 + 8.7 + 3.6, _n_.
  print 7.8 + 5.6 + 9.8 + 5.0, _n_.
  print 1.4 + 2.2 + 4.6 + 5.1, _n_.
  print 1.8 + 3.6 + 3.5 + 7.9, _n_.
  print 5.0 + 6.5 + 7.4 + 4.0, _n_.
  print 5.2 + 3.2 + 5.2 + 4.0, _n_.

  print 7.3 + 1.4 + 3.7 + 3.7, _n_.
  print 5.7 + 9.2 + 1.3 + 5.3, _n_.
  print 2.3 + 2.3 + 3.1 + 1.3, _n_.
  print 7.2 + 8.9 + 1.1 + 9.6, _n_.
  print 7.3 + 6.8 + 7.5 + 2.8, _n_.
  print 4.2 + 7.3 + 9.3 + 1.4, _n_.
  print 5.1 + 8.1 + 4.2 + 5.9, _n_.
  print 2.8 + 4.7 + 9.0 + 5.2, _n_.
  print 7.7 + 6.3 + 1.4 + 8.4, _n_.
  print 5.1 + 6.6 + 6.3 + 5.2, _n_.
  print 6.0 + 4.6 + 1.5 + 1.5, _n_.
  print 9.1 + 1.8 + 2.8 + 5.9, _n_.

}

