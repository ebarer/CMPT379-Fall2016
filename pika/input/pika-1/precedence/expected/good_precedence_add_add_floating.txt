20.2
11.3
21.3
20
14
11.3
21.5
17.7
22.8
17.1
22.6
10.5
16.3
17.6
21.9
21.4
17.3
9.7
15.2
17.3
16.1
8.8
17.3
14.3
16.1
23.7
19.2
14.7
16.1
26.8
22.3
28.2
13.3
16.8
22.9
17.6
16.1
21.5
9
26.8
24.4
22.2
23.3
21.7
23.8
23.2
13.6
19.6
